pytorch_lightning/trainer/trainer.py

```diff
@@ -1964,12 +1964,12 @@ def should_rank_save_checkpoint(self) -> bool:
         )
 
     @property
-    def _strategy_type(self) -> _StrategyType:
-        return self._accelerator_connector._strategy_type
+    def _strategy_type(self) -> Optional[str]:
```

Comment:
The base strategy does not define a strategy_name so it should be 

```suggestion
    def _strategy_type(self) -> str:
```

Reply:
This is for customized strategy, they may not define a name.